src/main/groovy/org/shipkit/gradle/ReleaseConfiguration.java

```diff
@@ -218,11 +222,12 @@ public String getWriteAuthToken() {
             if(envVar != null){
                 return envVar;
             }
-            throw new GradleException("Please export 'GH_WRITE_TOKEN' env variable first!\n" +
-                    "  The value of that variable is automatically used by Shipkit.\n" +
-                    "  It is highly recommended to keep write token secure and store env variable with your CI configuration.\n" +
-                    "  Alternatively, you can configure the write token explicitly:\n" +
-                    "    shipkit.gitHub.writeAuthToken = 'secret'");
+            LOG.lifecycle("  BEWARE! Shipkit.gitHub.writeAuthToken is unspecified.\n" +
```

Comment:
Let's check this in so that the build is happy. However, we should find a better solution very soon. I suspect that now the build will show this hearty warning with every invocation for most normal builds. It gives ungraceful experience to the end users of Shipkit (we will have lots of them soon :).